### ISSUE TITLE
 customized latex command not recognized

### ISSUE THREAD

**Issue by daihuichen** [arXiv/html_feedback#2115]
### Description

I think \Description is a customized latex command, but not recognized

### (Required) What is this issue most closely related to? Select one.

Other

### Internal issue ID

13440ca4-60e4-4079-a3a5-24a76ca0be94

### Paper URL

https://arxiv.org/html/2407.12117v1

### Browser

Chrome/127.0.0.0

### Device Type

Desktop

**Comment by dginev** [comment]
Bookkeeping comment: This is pending the upgrades from brucemiller/LaTeXML#2666

**Comment by dginev** [comment]
Thank you for your report - the specific issue is now solved.

We have regenerated the article and you should be able to see an improvement.
